apps/crn-server/test/fixtures/working-groups.fixtures.ts

```diff
@@ -50,7 +50,7 @@ export const getWorkingGroupDataObject = (): WorkingGroupDataObject => ({
 });
 
 export const getWorkingGroupResponse = (): WorkingGroupResponse =>
-  getWorkingGroupDataObject();
+  getWorkingGroupDataObject() as WorkingGroupResponse;
```

Comment:
i dont think this is true though? there is that inActive property marked as required that you added to leaders and members and this fixture doesn't have it

Reply:
So for all the fixtures we don't populate the leaders or member properties, so the type of `WorkingGroupResponse` is the same as `WorkingGroupObject`. Should i just copy getWorkingGroupDataObject function instead of calling it?